examples/gnrc_border_router/Makefile

```diff
@@ -0,0 +1,48 @@
+# name of your application
+APPLICATION = gnrc_networking
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+ifeq (,$(SLIP_UART))
+  # set default
+  SLIP_UART="(UART_NUMOF-1)"
+endif
+ifneq (,$(SLIP_BAUDRATE))
+  # set default
+  SLIP_BAUDRATE=115200
+endif
+
+CFLAGS += -I$(CURDIR)
+CFLAGS += -DSLIP_UART=$(SLIP_UART)
+CFLAGS += -DSLIP_BAUDRATE=$(SLIP_BAUDRATE)
+
+# Include packages that pull up and auto-init the link layer.
+# NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
+USEMODULE += gnrc_netif_default
+USEMODULE += auto_init_gnrc_netif
+# Include SLIP package for IP over Serial communication
+USEMODULE += gnrc_slip
+# Specify the mandatory networking modules for 6LoWPAN border router
+USEMODULE += gnrc_sixlowpan_border_router_default
+# Add a routing protocol
+USEMODULE += gnrc_rpl
```

Comment:
FIB is pulled in automatically and I thought the border router is supposed to be root